.github/workflows/build_eloquent.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
         echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable bionic main" > /etc/apt/sources.list.d/gazebo-stable.list
         wget https://packages.osrfoundation.org/gazebo.key -O - | apt-key add -
         sudo apt update
-        sudo apt install python3-shapely python3-yaml python3-requests \
+        sudo apt install python3-shapely python3-yaml python3-requests libopencv-dev \
```

Comment:
Looks like libopencv-dev can be installed via `rosdep` if we add it to the `package.xml` file?

Reply:
Gotcha, will try test it out the CI